frontend/src/components/NavBar/NavBar.module.css

```diff
@@ -23,9 +23,9 @@
 }
 
 .active {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
I was under the impression we would be using the same font for literally everything on the website?

Reply:
Note: I might be misunderstanding this because to my eye the font all looks the same in the screenshot but I could just be dumb

Reply:
yeah, make sure you're using the same standard font for the entire app